src/Views/Form/Templates/Sequoia/Sequoia.php

```diff
@@ -122,37 +123,69 @@ public function loadScripts() {
 		";
 		wp_add_inline_style( 'give-sequoia-template-css', $registrationCss );
 
-		$recurringDynamicCss = "
-			.give-recurring-donors-choice:hover,
-			.give-recurring-donors-choice.active {
-				border: 1px solid {$primaryColor};
-			}
-			.give-recurring-donors-choice input[type='checkbox'] + label::after {
-				background-image: url(\"data:image/svg+xml,%3Csvg width='15' height='11' viewBox='0 0 15 11' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M5.73047 10.7812C6.00391 11.0547 6.46875 11.0547 6.74219 10.7812L14.7812 2.74219C15.0547 2.46875 15.0547 2.00391 14.7812 1.73047L13.7969 0.746094C13.5234 0.472656 13.0859 0.472656 12.8125 0.746094L6.25 7.30859L3.16016 4.24609C2.88672 3.97266 2.44922 3.97266 2.17578 4.24609L1.19141 5.23047C0.917969 5.50391 0.917969 5.96875 1.19141 6.24219L5.73047 10.7812Z' fill='%23{$rawColor}'/%3E%3C/svg%3E%0A\");
-			}
-		";
-		wp_add_inline_style( 'give-sequoia-template-css', $recurringDynamicCss );
+		if ( Utils::isPluginActive( 'give-recurring/give-recurring.php' ) ) {
```

Comment:
@henryholtgeerts What do you think using only one property to collect all addon CSS and use only one

```php
<?php
if( $addonDynamicCss ){
    wp_add_inline_style( 'give-sequoia-template-css', $addonDynamicCss );
}
```

to register style

Reply:
Done, ref: https://github.com/impress-org/givewp/commit/b53771eecf80e58cc4c0dcfb061815702f8c7d2f